Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/Reflection/LegacyReflectionBridge.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Serialization/EditContext.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzFramework/DocumentPropertyEditor/Reflection/Attribute.h>
+#include <AzFramework/DocumentPropertyEditor/Reflection/Visitor.h>
+
+namespace AZ::Reflection
+{
+    //! These synthetic attributes are injected into legacy reflection data to give additional context
+    //! for use in the DocumentPropertyEditor.
+    namespace DescriptorAttributes
+    {
+        //! The UIHandler (or PropertyEditor Type) this property should use.
+        //! Type: String
+        extern Name Handler;
+        //! If specified, this property should have a label with the specified text.
+        //! Type: String
+        extern Name Label;
+        //! Specifies the JSON path to where this property would be located in a JSON serialized instance,
+        //! relative to the instance in a Visit call.
+        //! Type: String (can be parsed by AZ::Dom::Path)
+        extern Name SerializedPath;
```

Comment:
Not fully clear on what the path is relative to here. Better comments or better name would be helpful here.

Reply:
Slightly clarified the comment, but it's the Path relative to the root of whatever the thing you're reflecting via a Visit method, the intent is to use this later to help with the prefab adapter

Reply:
This should be marked as const as well.